src/porepy/examples/mandel_biot.py

```diff
@@ -1402,6 +1384,7 @@ def set_materials(self):
         """
         super().set_materials()
         self.exact_sol = MandelExactSolution(self)
+        self.results = []
```

Comment:
Are you sure this should go here? Init seems better than in set_materials, but other options could be possible. If a method does something which is not covered by its name, I try to check whether the operation can be moved or if more fundamental refactoring is necessary. Sometimes, pragmatism is called for, of course. Opinions, @keileg ?

Reply:
I moved the initialization of `.results` to the base solution strategy after shifting the functionality of `VerificationDataSaving` into `DataSavingMixin` and removing the former (as discussed in person). Now all models have it and it will be filled if the user provides a custom `collect_data` method.

Reply:
Sounds very reasonable